omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/PersonController2_2Test.java

```diff
@@ -75,7 +75,16 @@ public void shouldEditAPerson() throws Exception {
         assertTrue(person.isDead());
         assertNotNull(person.getCauseOfDeathNonCoded());
     }
-    
+    @Test
+    public void shouldEditAPersonBirthTime() throws Exception {
+        Person person = service.getPersonByUuid(getUuid());
+        String json = "{\"birthtime\": \"1970-01-01T20:20:00.000\"}";
+        SimpleObject response = deserialize(handle(newPostRequest(getURI() + "/" + getUuid(), json)));
+        assertNotNull(response);
+        Object responsePersonContents = PropertyUtils.getProperty(response, "person");
+        assertNotNull(responsePersonContents);
+        //assertEquals("1970-01-01T20:20:00.000+0000", PropertyUtils.getProperty(responsePersonContents, "birthtime").toString());
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
Sorry, I forgot to remove the commented part.

Reply:
please check once.